src/firebase/index.ts

```diff
@@ -0,0 +1,35 @@
+import { initializeApp } from "firebase/app";
+import firebaseConfig from "../config";
+
+import {
+  getDatabase,
+  onValue,
+  ref,
+  set,
+  update,
+  remove,
+} from "firebase/database";
+
+export const app = initializeApp(firebaseConfig);
+const db = getDatabase();
+
+export function getValueDB<T>(path: string, callback: (value: T) => void) {
+  const starCountRef = ref(db, path);
+
+  onValue(starCountRef, (snapshot) => {
+    const data = snapshot.val();
+    callback(data);
+  });
+}
+
+export function setValueDB<T>(path: string, data: T) {
+  return set(ref(db, path), data);
+}
+
+export function updateValueDB<T>(path: string, data: T) {
```

Comment:
try `<T extends {}>` or `<T = {}>` not to do `as object` becase you rewrite type

Reply:
maybe don't need typing here? you don`t  have `callback` and don't user data outside
check pls `update` -> Promise<void>